_component_design/switches.md

```diff
@@ -1,85 +1,105 @@
 ---
 title: Switches
```

Comment:
in general, I've been trying to use singular for the titles.  Tabs became tabset, Date Pickers became Date Picker, and so on.  Ryan took this approach on the helix-ui site, and I think it makes it clean.

Reply:
Updated to `title: switch` I left the md file and the images folder name in the plural because all other docs / folders follow the plural in the repo. Users won't know.